lib/features/identity_creator/presentation/identity_creator_controller.dart

```diff
@@ -782,15 +785,50 @@ class IdentityCreatorController extends BaseController with DragDropFileMixin {
     });
   }
 
+  Future<void> onPasteImageSuccess(
+    BuildContext context,
+    List<FileUpload> listFileUpload,
+    {required double maxWidth}
+  ) async {
+    await _uploadMultipleFilesToPublicAsset(
+      context,
+      listFileUpload.toListFileInfo(),
+      maxWidth: maxWidth);
+  }
+
+  void onPasteImageFailure(
+    BuildContext context,
+    List<FileUpload>? listFileUpload,
+    {String? base64,
+    required UploadError uploadError}
+  ) {
+    logError('$runtimeType::onPasteImageFailure: $uploadError');
+    appToast.showToastErrorMessage(
+      context,
+      AppLocalizations.of(context).thisImageCannotBePastedIntoTheEditor);
+  }
+
   void onLocalFileDropZoneListener({
     required BuildContext context,
     required DropDoneDetails details,
     required double maxWidth
   }) async {
-    try {
-      clearFocusEditor(context);
+    clearFocusEditor(context);
+
+    final listFileInfo = await onDragDone(context: context, details: details);
+    if (!context.mounted) return;
```

Comment:
We should not block upload when context is not mounted. Because upload process is not bound by context. Context only affects view and here is toast

Reply:
Done